### ISSUE TITLE
Component folders are suggested when opening a function's braces

### ISSUE THREAD

**Issue by chivalry** [forcedotcom/salesforcedx-vscode#3902]
### Component folders are suggested when opening a function's braces.

Defining a function by typing up to the opening brace shows this:

![image](https://user-images.githubusercontent.com/600992/157544964-29859766-f842-490f-8d00-570fe232818d.png)

### Steps To Reproduce:

1.  In a Lightning Web Component's JavaScript file, within the class definition type `@api myFunc() {`.

### Expected result

I expect absolutely nothing to happen.

### Actual result

I'm presented with IntelliSense offerings of component "folders" (as namespaces by Salesforce).

### Additional information

I'm able to prevent this behavior by unchecking the option for IntelliSense to show folders.

![image](https://user-images.githubusercontent.com/600992/157547902-81bc9a39-8c1c-4f42-b3ed-a01eb7430d42.png)

**Salesforce Extension Version in VS Code**: 54.4.0

**SFDX CLI Version**: 7.132.0

**OS and version**: macOS 12.2.1

**Comment by ndrewr** [comment]
Seconded.
This happens when creating a conditional block as well:

![image](https://user-images.githubusercontent.com/9262383/157555508-1e6b18a1-67d3-46ff-9414-3fb2d6195574.png)

**Comment by randi274** [comment]
Thanks for getting this filed; we introduced some [new functionality](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode/CHANGELOG.md#salesforcedx-vscode-lightning--salesforcedx-vscode-lwc) to add autocompletion for attributes on the html side, and were a little overly-inclusive about should trigger autocompletion, and here it's extraneous. I've identified the root cause, and should have a fix up for it soon. Stay tuned!
